tests/templates/test_embeddings/test_angle.py

```diff
@@ -276,6 +276,32 @@ def test_jax(self, tol):
 
         assert np.allclose(grads[0], grads2[0], atol=tol, rtol=0)
 
+    @pytest.mark.jax
+    def test_jax_jit(self, tol):
+        """Tests jit with the jax interface."""
+
+        import jax
+        import jax.numpy as jnp
+
+        features = jnp.array([1.0, 1.0, 1.0])
+
+        dev = qml.device("default.qubit", wires=3)
+
+        circuit = jax.jit(qml.QNode(circuit_template, dev))
+        circuit2 = jax.jit(qml.QNode(circuit_decomposed, dev))
+
+        res = circuit(features)
+        res2 = circuit2(features)
+        assert qml.math.allclose(res, res2, atol=tol, rtol=0)
```

Comment:
We should compare the results with the `non-jit` version as well. (might be applicable to other tests as well)